accounts-db/src/accounts_db.rs

```diff
@@ -6005,7 +6005,7 @@ impl AccountsDb {
         // with this function call. This means, if we get into this case, we can be
         // confident that the entire state for this slot has been flushed to the storage
         // already.
-        let mut scan_storages_elasped = Measure::start("scan_storages_elasped");
+        let mut scan_storages_elapsed = Measure::start("scan_storages_elapsed");
```

Comment:
This is a variable name. If changed, it also needs to be changed at the call site.